.changeset/chilly-owls-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/node": minor
+---
+
+Added the `isIndependent: true` option to the `NodeJS` adapter. With this option, when building, the output folder `dist/server` will contain all the necessary data to start the server. And your server will be able to work without the `node_modules` folder
```

Comment:
```suggestion
Added the `isIndependent: true` option. With this option, when building, the output folder `dist/server` will contain all the necessary data to start the server. And your server will be able to work without the `node_modules` folder
```

The package `@astrojs/node` is already the adapter, I think it's redundant to say "to the Node.js adapter".

Reply:
I plan to move this to experimental config: 
```
export default defineConfig({
	output: 'server',
	adapter: node({
		mode: 'standalone',
	}),
	experimental: {
		isIndependent: true,
	}
});
```

Reply:
In this PR?

Reply:
Yes, I have now moved this option to `config.experimental.isIndependent`. Without this change, building without `node_modules` will not work for some frameworks